alternator/executor.cc

```diff
@@ -728,12 +784,42 @@ static future<bool> is_view_built(
 
 }
 
-static future<rjson::value> fill_table_description(schema_ptr schema, table_status tbl_status, service::storage_proxy& proxy, service::client_state& client_state, tracing::trace_state_ptr trace_state, service_permit permit)
+future<> executor::cache_newly_calculated_size_on_all_shards(schema_ptr schema, std::uint64_t size_in_bytes, std::chrono::nanoseconds ttl) {
+    auto expiry = std::chrono::high_resolution_clock::now() + ttl;
```

Comment:
I still don't understand why you don't want to have describe_table_info_manager.now() like I suggested, and instead you want to "guess" that the right clock is high_resolution_clock.

By the way, I would put expiry inside the inner block below - why calculate it outside and then pass it into the lambda, instead of passing ttl into the lambda?